client/actions/assignments/notifications.js

```diff
@@ -307,14 +315,39 @@ const onAssignmentRemoveFailed = (_e, data) => (
     }
 );
 
+const onAssignmentDeleted = (_e, data) => (
+    (dispatch, getState, {notify}) => {
+        const sessionId = selectors.general.sessionId(getState());
+
+        if (get(data, 'items.length', 0) > 0 && get(data, 'session') !== sessionId) {
+            const msg = data.items.map((i) => gettext('{{ type }} assignment \'{{ slugline }}\' is deleted',
+                {
+                    type: get(i, 'type'),
+                    slugline: get(i, 'slugline'),
+                })).join('\n');
+
+            notify.warning(msg);
+            data.items.forEach((item) => {
+                dispatch(_notifyAssignmentEdited(item.id, data.session));
+            });
+
+            // Load all assignment groups as the assignment deleted can be in any state
+            dispatch(assignments.ui.reloadAssignments());
```

Comment:
You could be working with 2 tabs open (one for assignments and other for planning). You might want to check workspace === assignments and reloadassignments.